source/operations/concepts/erasure-coding.rst

```diff
@@ -35,7 +35,7 @@ Erasure Sets
 ------------
 
 An *Erasure Set* is a group of drives onto which MinIO writes erasure coded objects.
-MinIO randomly and uniformly distributes the data and parity blocks of a given object across the erasure set drives, where a given drive has no more than one block of either type per object (no overlap).
+MinIO randomly and uniformly distributes the data and parity shards of a given object across the erasure set drives, where a given drive has no more than one block of either type per object (no overlap).
```

Comment:
Not sure where the best place is but probably mentioning that we don't care whether a shard is parity or data, as long as we have quorum we can rebuilt since conceptually this catches many people off guard

Reply:
Found a spot for it